gammapy/maps/wcs.py

```diff
@@ -594,6 +596,30 @@ def upsample(self, factor):
     def to_slice(self, slices):
         raise NotImplementedError
 
+    def solid_angle(self):
+        """ Returns solid angle array (in `sr`) as `~astropy.units.Quantity`
```

Comment:
Suggest to mention shape of returned array in an extra line. I.e. docs shoud say if this is 2D or higher dimensional if there are extra axes.